static/styles/message.css

```diff
@@ -0,0 +1,128 @@
+                grid-row: 1 / 2;
+
+                > span {
+                    display: flex;
+                }
+
+                .inline_profile_picture {
+                    flex-shrink: 0;
+                    /* Let user profile take extra height without
+                       having any affect on height of the container. */
+                    position: absolute;
+                    margin-top: 6px;
+                }
+
+                .sender_name {
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    /* It is important to use line-height `normal` here since user's name can
+                       be in any language and `line-height: 1` doesn't work to accommodate text
+                       from start and end vertically in all languages. */
+                    line-height: normal;
+                    /* Add padding to align user name with the content */
+                    padding-left: $first_column_width;
+                    outline: none;
+                    margin-top: 3px;
+                }
+            }
+        }
+    }
+
+    .messagebox .messagebox-content {
+        display: grid;
+        align-items: start;
+        padding-left: 10px;
+        padding-right: 2px; /* For message focused border */
+        grid-template-rows: repeat(4, auto);
+        /* 60px 64px were chosen to fit message controls and time visually.
+           It is easy to adjust them in future if need be since their horizontal placement
+           will be handled by them automatically using `justify-self` property. */
+        grid-template-columns: $first_column_width auto 60px 64px;
+
+        .message_controls {
+            grid-row-start: 1;
+            grid-column-start: 3;
+            line-height: 1;
+            justify-self: end;
+            position: relative;
+            top: 6px;
+        }
+
+        .message_edit_notice {
+            grid-row-start: 1;
+            position: relative;
+            top: $distance_of_text_elements_from_message_box_top;
+        }
+
+        .message_time {
+            line-height: 1;
+            justify-self: center;
+            grid-row-start: 1;
+            grid-column-start: 4;
+            position: relative;
+            top: $distance_of_text_elements_from_message_box_top;
+        }
+
+        .message_content {
+            grid-row-start: 1;
+            grid-column-start: 2;
+            padding: 2px;
+        }
+
+        .message_reactions {
+            grid-row-start: 4;
+            grid-column-start: 2;
+        }
+
+        .message_edit {
+            grid-row-start: 2;
+            grid-column-start: 2;
+        }
+
+        .alert-msg {
+            grid-row-start: 1;
+            grid-column: 3 / 5;
+            margin-top: 4px;
+        }
+
+        .message_length_controller {
+            grid-row-start: 3;
+            grid-column-start: 2;
+        }
+    }
+}
```

Comment:
What do you think about calling this `message_feed.css`, and planning for it to include other CSS that makes up the message feed (stuff like recipient bars) that are not the message body (aka `rendered_markdown.css`)?

I think this name, or perhaps `message_row.css`, are reasonable if we're not going to include some of the CSS for the rest of the message feed.

Reply:
Maybe `message_row.css` is a good scope for a file; there's a lot of things that should probably be moved here from `zulip.css` even with that scope.

Reply:
Yeah, `message_row.css` makes sense to me. Renamed.